src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -439,8 +439,9 @@ public static string SheetsNotFound(List<string> listOfSheetNames)
             return sb.ToString();
          }
 
-         public static readonly string UseFiltersForImport = "Use the filters for importing the data";
-         
+         public static readonly string UseFiltersForImport = "Use filters for importing data";
+         public static readonly string UseFiltersForImportTooltip = "When checked, the data comming from the file will also be filter by the importer. When unchecked, the filters only affect this view. Check documentation for more information on defining filters: <href=https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/features-of-tables#filtering>https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/features-of-tables#filtering</href>";
```

Comment:
_When checked, the data comming from the file will also be filter by the importer._
Would reformulate this part to something like:
"When checked, the filter will be applied to the data during the import process"

Also maybe it's better to use "selected" and "deselected" instead of "checked" and "unchecked"?